_posts/2018-05-16-cockpit-168.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Cockpit 168
+author: pitti
+date: 2018-05-16 11:00
+tags: cockpit linux
+slug: cockpit-168
+category: release
+summary: Cockpit with improved root privilege check
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly.
+Here are the release notes from version 168.
+
+### Improve checks for root privilege availability
+
+Many actions, like rebooting the machine or joining an IPA domain, can only be
+done by an administrator (`root`). To decide whether to enable these actions,
+Cockpit previously checked if the user was a member of the `wheel`
+(RHEL/Fedora) or `sudo` (Debian) groups.
+
+This was insufficient: Other operating systems or local customizations might
+use other groups, or have user instead of group based sudo rules. For example
+with FreeIPA central user management, sudo access is given through the
+`admins@domain.name` group.
```

Comment:
Cockpit previously checked if the user was a member of the `wheel` (used in
Red Hat Enterprise Linux, CentOS, and Fedora) or `sudo` (used in Debian and
Ubuntu) groups to enable these actions. Simple group checking was insufficient,
as different group names are used by other operating systems and configurations
or a system might be set up to rely on custom user-based sudo rules.